publications/abstracts/reservoir-computing/acknowledgements.tex

```diff
@@ -0,0 +1,14 @@
+\section{Acknowledgments}
```

Comment:
Please use the up-to-date acks in https://docs.google.com/spreadsheets/d/1VybV0oMPiqpInTwgO0ICq0EC4AL_fGj-zbXGJAhRiVU/edit?usp=sharing